### PR TITLE
Db fairs

### PR DESCRIPTION
**Pull request by Joanartdacosta**
- create reducer fairSlice, action saveAllFairs and include reducer in store
- load all fairs from mongodb
- fix fairs filter

### CODE REVIEW

File: app/components/dashboard/fairs/filter_fair/FilteringFair.js [hunk 1]
```diff
@@ -29,17 +31,30 @@ export default function FilteringFairByDateAndRegion() {
...
-    return fairs.filter((fair) => {
-      if (fair.endDate < startDate) {
+  function filterFairsByDate(startDate, endDate, current) {
+    return current.filter((fair) => {
+      const dateObjStartDate = new Date(startDate);
+      const dateObjEndDate = new Date(endDate);
+      const dateObjFairStartDate = new Date(fair.date.endDate);
+      const dateObjFairEndDate = new Date(fair.date.startDate);
+
+      console.log(current);
```
Comment (doublethink13) [inline app/components/dashboard/fairs/filter_fair/FilteringFair.js:47]: console log

File: app/components/dashboard/fairs/filter_fair/FilteringFair.js [hunk 2]
```diff
@@ -29,17 +31,30 @@ export default function FilteringFairByDateAndRegion() {
...
+
+      if (fair._id === "65b40904a7e4271800c4071b") {
+        console.log(dateObjFairEndDate);
```
Comment (doublethink13) [inline app/components/dashboard/fairs/filter_fair/FilteringFair.js:50]: console log

File: app/components/dashboard/fairs/filter_fair/FilteringFair.js (hunk 1 above)
Comment (Joanartdacosta) [reply app/components/dashboard/fairs/filter_fair/FilteringFair.js]: my bad, sorry! deleted

File: app/components/dashboard/fairs/filter_fair/FilteringFair.js (hunk 2 above)
Comment (Joanartdacosta) [reply app/components/dashboard/fairs/filter_fair/FilteringFair.js]: my bad, sorry! deleted

**Pull request merged**